src/domain/session/room/RoomViewModel.js

```diff
@@ -283,10 +283,16 @@ export class RoomViewModel extends ViewModel {
             console.error(err.stack);
         }
     }
-    
+
     get composerViewModel() {
         return this._composerVM;
     }
+
+    toggleDetailsPanel() {
```

Comment:
the implementation looks like it will just open rather than toggle? Change name?